gax-java/gax-grpc/src/test/java/com/google/api/gax/grpc/GrpcCallContextTest.java

```diff
@@ -78,7 +78,7 @@ void testWithCredentials() {
     GrpcCallContext emptyContext = GrpcCallContext.createDefault();
     assertNull(emptyContext.getCallOptions().getCredentials());
     GrpcCallContext context = emptyContext.withCredentials(credentials);
-    assertNotNull(context.getCallOptions().getCredentials());
+    assertNull(context.getCallOptions().getCredentials());
```

Comment:
Can you add a comment above this test to explain why this is null now?

The Credentials will be passed to the channel when the channel is built, etc.

Reply:
Added a comment.